### DISCUSSION TITLE
delete notes from the editing or viewing view

### DISCUSSION THREAD

**Discussion by JJJHolscher** [damionx7/Zettel-Notes-Documentation#129] [Feature Requests]
As simple as the title says, but if you like reading then I'll elaborate.

As I understand, one can only delete notes in the repository view.
But say I follow a couple links when browsing my zettels and suddenly find a note that can be deleted, then I'd like to be able to do that without having to re-search for it in the repository view.
Maybe long-pressing the file name in the top should give you the same options for that note as one would get when long pressing the note in the repo view, or have the delete option be added under the i button.

I see a lotalot of feature requests on here, >50% of them are probably served best with their own plugin (please don't die due to feature bloat, your app is an absolute gem as-is). I don't think it makes a lot of sense to make a plugin for something as basic as this, but I **might** if this keeps annoying me for long enough.

**Comment by damionx7** [comment]
Hey. Thank you for the appreciation. I think delete option can be added in note info pop up. Note info can be accessed while viewing the note.

**Reply by JJJHolscher** [reply]
With "can be accessed" you mean, "were you (JJJHolscher) to make a plugin it's a good idea to add it there" right?
Since I found no such button at what I expect to be the note info pop up.

I checked in the note view
![Screenshot_20240713-175736](https://github.com/user-attachments/assets/e6ee8946-a231-487f-8cea-5c2a1875d838)
and when pressing the i icon on top
![Screenshot_20240713-175744](https://github.com/user-attachments/assets/f3c81ef7-f3ce-4954-81ff-dbbf178e3f0b)
I don't see that option, nor can I scroll here

When in the repository/folder/main view
![Screenshot_20240713-175704](https://github.com/user-attachments/assets/a1f22150-f49c-48a3-88c9-7cf629c36ddc)
I see the option when long-pressing or pressing the triple dots of a specific note
![Screenshot_20240713-175716](https://github.com/user-attachments/assets/08480443-e61e-40ef-997c-160d1f4d645b)
But that's not the (only) place I want to have that option.

Or do you mean there is some setting that makes such a delete button show up in the second screenshot? Since I looked and didn't find one.

**Reply by damionx7** [reply]
I meant that I will add this option in upcoming versions. Sorry for the confusion. 😀

**Comment by damionx7** [comment]
I have added this feature.
